src/components/session/LevelFilter/index.tsx

```
@@ -0,0 +1,31 @@
+export default function LevelFilter({ value, onChange }: LevelFilterProps) {
+  return (
+    <Dropdown size="lg" hasSelected={Boolean(value)}>
+      <Dropdown.Trigger>
+        {value ? getOptionLabel(LEVEL_OPTIONS, value) : '난이도'}
+      </Dropdown.Trigger>
+      <Dropdown.Content>
+        {LEVEL_OPTIONS.map(({ label, value }) => (
+          <Dropdown.Item
+            key={value ?? 'all'}
+            selected={value === value}
+            onSelect={() => onChange(value)}
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)